src/discord.cpp

```diff
@@ -30,8 +30,6 @@
 // Uncomment this to provide some additional qDebug() output:
 // #define DEBUG_DISCORD 1
 
-QReadWriteLock Discord::smReadWriteLock;
```

Comment:
Are you absolutely sure that this is not needed to protect the `static` elements that get updated by the external, dynamically loaded `discord-rpc` library when it calls the `handleDiscordReady(...)` function in response to the timer that runs the library's `Discord_RunCallbacks()` function every 50 mSec once the library was loaded.

Reply:
I have taken your comment https://github.com/Mudlet/Mudlet/pull/4176#issuecomment-713485176 on-board so am prepared to give things a spin without this...